docs/userguides/testing.md

```diff
@@ -396,6 +396,47 @@ def test_unauthorized_withdraw(contract, hacker):
         contract.withdraw(sender=hacker)
 ```
 
+You can also use the ABI of the error if you don't have access to an instance:
+
+```python
+import ape
+
+def test_unauthorized(contract, hacker, project):
+    with ape.reverts(project.MyOtherContract.Unauthorized2, addr=hacker.address):
```

Comment:
Just use full `address` for the kwarg

Edit: or wait, is this the event arg?

Reply:
ya that is a custom kwarg, i can use a different example though, and this was kinda copied from the example above so nothing fresh here, can be revised

Reply:
you dont actually assert the address of the contract, that happens via the instance of the contract it is coming off of (see one of the comments i added in that last commit regarding this).

Reply:
changed to `unauth_address`